src/xdocs/config_annotation.xml

```diff
@@ -159,14 +166,14 @@ public String getNameIfPresent() { ... }
 &lt;/module&gt;
         </source>
         <p>
-          Example to allow one single parameterless annotation on the same line
+          Example to allow multiple annotations on the same line
```

Comment:
like above, I think "single parameterless" should be kept. "multiple" is ok.

Reply:
I have kept both. The line numbers are changed because I have added the default configuration at the top.